ckanext/emailasusername/plugin.py

```diff
@@ -88,3 +93,25 @@ def user_emails_match(key, data, errors, context):
         )
     else:
         data[('email',)] = email1
+
+
+@classmethod
+def search(cls, querystr, sqlalchemy_query=None, user_name=None):
```

Comment:
there's no indentation in this method. What class is this method for supposed to be?

Reply:
Yeah welcome your thoughts on this.  I'm overriding a class method with this function and the decorator was necessary to get it to be usable as `CLS.search(querystr ...)` How would you do it?